src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -202,7 +207,7 @@ protected async Task WaitForExecutionOnRelevantNodes(JsonOperationContext contex
 
             try
             {
-                using (var cts = CancellationTokenSource.CreateLinkedTokenSource(ServerStore.ServerShutdown))
+                using (var cts = CancellationTokenSource.CreateLinkedTokenSource(ServerStore.ServerShutdown, HttpContext.RequestAborted))
```

Comment:
use `CreateHttpRequestBoundOperationToken`

Reply:
I cant..
I need cts because I am using the 'cts.CancelAfter(...)' right after that..